statsmodels/tsa/statespace/sarimax.py

```diff
@@ -523,6 +523,15 @@ def __init__(self, endog, exog=None, order=(1, 0, 0),
         if k_diffuse_states == 0:
             self.initialize_stationary()
 
+        # update _init_keys attached by super
+        self._init_keys += ['order',  'seasonal_order', 'trend',
+                 'measurement_error', 'time_varying_regression',
+                 'mle_regression' 'simple_differencing',
```

Comment:
No, that's the first, I forgot to commit this change, still the 22 failures (using data.orig_endog) 
will push now the missing commit.